docs/src/man/structure.md

```diff
@@ -20,13 +20,14 @@ The `Bio.Structure` module provides functionality to manipulate macromolecular s
 To download a PDB file:
```

Comment:
Perhaps change this section title from "Parsing PDB files" to "Basics" as this makes more sense in the new arrangement.